### DISCUSSION TITLE
Users and Organization

### DISCUSSION THREAD

**Discussion by kkounouni** [GetDKAN/dkan#3831] [Ideas]
Hello, I would like to assign users/accounts in the Organization in order they have access only to their datasets and upload the dataset only to specific organizations. Is it possible in DKAN2?

**Comment by janette** [comment]
@kkounouni install the [group](https://www.drupal.org/project/group) module and configure as needed.

**Comment by muhammadmohsinali** [comment]
Hi @kkounouni I recently done this activity. Install group module and configure it. Add reference field of group in data content type. Just hide the schema publisher field via CSS properties. Add jQuery to auto select the dropdown value of group field to schema json field. This is the most appropriate way I found and it works perfectly.
Thanks

**Comment by kkounouni** [comment]
Thanks, @muhammadmohsinali . 

I have it in mind.

**Comment by dafeder** [comment]
Hello @kkounouni and @muhammadmohsinali, I'm wondering if you got anywhere with this. I know there was a bit of additional work needed to get the functionality and integration you were looking for.
